docs/usage.md

```diff
@@ -95,6 +95,28 @@ genome: 'GRCh37'
 
 You can also generate such `YAML`/`JSON` files via [nf-core/launch](https://nf-co.re/launch).
 
+### Decontanmination
+
+Filter the output files from metagenomics classifiers like `Kraken2`, `Centrifuge`, or `DIAMOND` to remove false positives and background contamination can be activated with `--decontamination`. This step compares results to the negative control to identify likely present species based on user-defined thresholds, effecitively eliminating background noise from microbiome species.
```

Comment:
```suggestion
Filter the output files from metagenomics classifiers like `Kraken2`, `Centrifuge`, or `DIAMOND` to remove false positives and background contamination can be activated with `--decontamination`. This step compares results to the negative control to identify likely present species based on user-defined thresholds. 
```

Reply:
This step would remove known contaminants from kits, right? Maybe mention that here?